lib/cob_index/macros/custom.rb

```diff
@@ -416,9 +416,9 @@ def normalize_issn
 
       def normalize_lccn
         Proc.new do |rec, acc|
-          orig = acc.dup
-          acc.map! { |x| StdNum::LCCN.normalize(x) }
-          acc << orig
+          acc.map! { |x|
+            formatted_x = x.gsub("#", " ")
+            StdNum::LCCN.normalize(formatted_x) }
           acc.flatten!
```

Comment:
You don't  need acc.flatten! anymore if you're removing `acc << orig` on line 421